101-route-server-quagga/azuredeploy.json

```diff
@@ -0,0 +1,467 @@
+                "description": "The base URI where artifacts required by this template are located. For example, if stored on a public GitHub repo, you'd use the following URI: https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-route-server-quagga/"
+            },
+            "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-route-server-quagga/"
+        },
+        "_artifactsLocationSasToken": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The sasToken required to access _artifactsLocation.  If your artifacts are stored on a public repo or public storage account you can leave this blank."
+            },
+            "defaultValue": ""
+        }
+    },
+    "variables": {
+        "location": "[parameters('location')]",
+        "vnetName": "[parameters('vnetName')]",
+        "addressSpace1Prefix": "[parameters('addressSpace1Prefix')]",
+        "subnet1Name": "[parameters('subnet1Name')]",
+        "subnet1Prefix": "[parameters('subnet1Prefix')]",
+        "subnet2Name": "[parameters('subnet2Name')]",
+        "subnet2Prefix": "[parameters('subnet2Prefix')]",
+        "subnet3Name": "[parameters('subnet3Name')]",
+        "subnet3Prefix": "[parameters('subnet3Prefix')]",
+        "subnet4Name": "[parameters('subnet4Name')]",
+        "subnet4Prefix": "[parameters('subnet4Prefix')]",
+        "gatewaysubnetPrefix": "[parameters('gatewaysubnetPrefix')]",
+        "rsName": "[parameters('routeServerName')]",
+        "peering1IP": "[parameters('peering1IP')]",
+        "peering1ASN": "[parameters('peering1ASN')]",
+        "rsBGPconnectionName": "[parameters('rsBGPconnectionName')]",
+        "subnet_rs1_Id": "[resourceId( 'Microsoft.Network/virtualNetworks/subnets', variables('vnetName'),'RouteServerSubnet' )]",
+        "vHubIPconfigName": "[parameters('vHubIPconfigName')]",
+        "bgpConnectionName": "[concat(variables('rsName'), '/', variables('rsBGPconnectionName'))]",
+        "vmName": "[parameters('vmName')]",
+        "nicName": "[concat(variables('vmName'),'-nic')]",
+        "vmPubIPName": "[concat(variables('vmName'),'-pubIP')]",
+        "vmPrivIP": "[parameters('vmPrivIP')]",
+        "publisher": "[parameters('publisher')]",
+        "offer": "[parameters('offer')]",
+        "sku": "[parameters('sku')]",
+        "vmSize": "[parameters('vmSize')]",
+        "nsgName": "[parameters('nsgName')]",
+        "scriptFolder": "scripts",
+        "customScriptFilePath": "[uri(parameters('_artifactsLocation'), concat(variables('scriptFolder'), '/', 'quaggadeploy.sh', parameters('_artifactsLocationSasToken')))]"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Network/networkSecurityGroups",
+            "name": "[variables('nsgName')]",
+            "apiVersion": "2020-06-01",
+            "location": "[variables('location')]",
+            "comments": "network security group",
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties

Reply:
done